Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -410,6 +410,11 @@ function PowerGenerator_JobComplete( j )
 end
 
 function LandingPad_Temp_UpdateAction(furniture, deltaTime)
+    
+    if(furniture.tile.room.ID ~= 0) then
+        return
+    end
+    
 	spawnSpot = furniture.GetSpawnSpotTile()
```

Comment:
For some reason it appears that this file uses tabs. Are we going for tabs in lua files?